docs/content/en/boilerplates/prerequisites.md

```diff
@@ -13,7 +13,10 @@ Deploy a kubernetes cluster using kurator's scripts.
 $ hack/local-dev-setup.sh
 reparing kind config in path: /tmp/tmp.xxxxx
 ...
-Local clusters is running.
+Local clusters is running. Note that the kind version and node image version need to be matched. 
+Now that kurator's local_dev_setup.sh is using the 1.25.3 node image. 
+You can refer to kind's release info(https://github.com/kubernetes-sigs/kind/releases).
```

Comment:
Can you be more specific, if there is a mismatch maybe we need to have a support matrix like this https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases

Reply:
Kurator now uses the kindest/node image from version 1.25.3, which works fine with king versions 0.16.0-0.19.0. It does not work with version 0.20.0 and 0.15.0 and earlier.